odm_sdk/scripts/delete_study_or_template.py

```diff
@@ -27,13 +26,13 @@ def main():
         connection.application('genestack/study-metainfo-editor').invoke('wipeStudy', accession)
         print(colored("Success", GREEN))
     except GenestackServerException as e:
-        p = re.compile('File .* not found')
-        result = p.search(e.message)
-        if result is not None:
-            print(colored("Study/template with accession %s does not exist" % accession,
-                          RED))
-        else:
-            print(colored(e, RED))
+        message = e
+        if e.stack_trace is not None:
+            p = re.compile(r"GenestackRestApiException: (.*?)(?=\n)")
+            result = p.search(e.stack_trace)
+            if result:
+                message = result.group(1)
+        print(colored(message, RED))
```

Comment:
Shouldn't it exit with code 1 in case of an error?

```suggestion
        print(colored(message, RED), file=sys.stderr)
        return sys.exit(1)
```

Reply:
Seems like we do so in other scripts, so yes, I can add